packages/angular/src/scatter-chart.component.ts

```diff
@@ -0,0 +1,37 @@
+import {
+	Component,
+	AfterViewInit
+} from "@angular/core";
+
+import { BaseChart } from "./base-chart.component";
+
+import { ScatterChart } from "@carbon/charts";
+
+/**
+ * Wrapper around `ScatterChart` in carbon charts library
+ *
+ * Most functions just call their equivalent from the chart library.
+ */
+@Component({
+	selector: "n-scatter-chart",
```

Comment:
Totally never caught this ... we'll want to switch the prefix to either `ibm-` (to match carbon-angular) or `c-` (for carbon) ... the easiest way going forwards will be `"ibm-scatter-chart, n-scatter-chart"` (or whatever prefix) until we hit v1.

Doesn't stop this PR, but just something we should sort out sooner rather than later.